Makefile

```diff
@@ -0,0 +1,37 @@
+SHELL = /bin/bash
+
+DOCKER_HOST = $(shell echo $$DOCKER_HOST)
+BUILD_TAG ?= git-$(shell git rev-parse --short HEAD)
+SHORT_NAME ?= telegraf
```

Comment:
whoops good catch. i had to copy this from telegraf since i lost it in the rebase